CHANGELOG.md

```diff
@@ -4,6 +4,13 @@
 
 - Update the dangerfile
 
+## [2.0.0] 2019-10-11
```

Comment:
I don't see an actual release that matches this.

Reply:
I assume this will be released on merge - generally wouldn't see the release until it's merged in master

Reply:
I usually release soon after merging.